tests/tests.yaml

```diff
@@ -205,34 +205,36 @@ answer_tests:
 
 other_tests:
   unittests:
+     # keep in sync with nose_ignores.txt
```

Comment:
this is the only real new line, otherwise I just sorted the arguments in alphabetical order and made the style more consistent